app/domain/authentication/authn_jwt/validate_status.rb

```diff
@@ -0,0 +1,124 @@
+module Authentication
+  module AuthnJwt
+
+    ValidateStatus = CommandClass.new(
+      dependencies: {
+        create_signing_key: Authentication::AuthnJwt::SigningKey::CreateSigningKeyFactory.new,
+        fetch_issuer_value: Authentication::AuthnJwt::ValidateAndDecode::FetchIssuerValue.new,
+        identity_from_decoded_token_provider_class: Authentication::AuthnJwt::IdentityProviders::IdentityFromDecodedTokenProvider,
+        validate_webservice_is_whitelisted: ::Authentication::Security::ValidateWebserviceIsWhitelisted.new,
+        validate_role_can_access_webservice: ::Authentication::Security::ValidateRoleCanAccessWebservice.new,
+        validate_webservice_exists: ::Authentication::Security::ValidateWebserviceExists.new,
+        validate_account_exists: ::Authentication::Security::ValidateAccountExists.new,
+        logger: Rails.logger
+      },
+      inputs: %i[authenticator_status_input enabled_authenticators]
+    ) do
+      extend(Forwardable)
+      def_delegators(:@authenticator_status_input, :authenticator_name, :account,
+                     :username, :status_webservice, :service_id, :client_ip)
+
+      def call
+        @logger.info(LogMessages::Authentication::AuthnJwt::ValidatingJwtStatusConfiguration.new)
+        validate_generic_status_validations
+        validate_secrets
+        @logger.info(LogMessages::Authentication::AuthnJwt::ValidatedJwtStatusConfiguration.new)
+      end
+
+      private
+
+      def validate_generic_status_validations
```

Comment:
This block feels common to all our authenticators.  In the future, are we considering moving this to a new class which can be passed to the `call` method to perform the check for any authenticator?

Reply:
Im working on refactor for the AuthenticationParameters mechanisem so lets see if this point relevant after the refactor